src/audio/multiband_drc/multiband_drc.c

```diff
@@ -328,6 +337,12 @@ static int multiband_drc_init(struct processing_module *mod)
 	}
 	multiband_drc_reset_state(&cd->state);
 
+	/* Initialize to enabled as a workaround for IPC4 kernel version 6.6
+	 * and before where the processing is never enabled via switch
+	 * control. The new kernel sends the IPC4 switch control and sets
+	 * this to the desired state before prepare.
+	 */
+	multiband_drc_process_enable(&cd->process_enabled);
```

Comment:
why was this moved?

Also your commit message is out of date, please update it